test_env_vars.yaml

```diff
@@ -5,12 +5,17 @@ api_url: http://localhost:8080/api/v1
 api_auth_type: null
 # Package that will be used for testing
 package:
-  # The URL for the upstream git repository
-  repo: https://github.com/release-engineering/retrodep.git
-  # A git reference at the given git repository
-  ref: e1be527f39ec31323f0454f7d1422c6260b00580
-  # Package managers
-  pkg_managers: ["gomod"]
+  golang:
+    # The URL for the upstream git repository
+    repo: https://github.com/release-engineering/retrodep.git
+    # A git reference at the given git repository
+    ref: e1be527f39ec31323f0454f7d1422c6260b00580
+    # Package managers
```

Comment:
I think this comment can be removed since the key name already makes this clear.